proc_IR.py

```diff
@@ -11,27 +11,35 @@
 clock_correction = 1.785
 filter_bandwidth = 5e3
 coh_sel = {'ph1':0,
-        'ph2':1}
+        'ph2':0}
 coh_err = {'ph1':1,# coherence channels to use for error
         'ph2':r_[0,2,3]}
 # }}}
 
 for searchstr,exp_type,nodename, postproc in [
-        ('200212_IR_3_30dBm', 'test_equip', 'signal', 
-            'spincore_IR_v1'),
+        #('w8_200731', 'test_equip', 2, 
+        #    'ag_IR2H'),
+        ('w8_200309', 'test_equip',2,'ag_IR2H'),
         ]:
     fl.basename = searchstr
-    s = find_file(searchstr, exp_type=exp_type,
+    if postproc=='ag_IR2H':
+        s = find_file(searchstr,exp_type=exp_type,
+                expno=nodename,
+                postproc=postproc, lookup=postproc_dict,
+                dimname='indirect')
+    if postproc=='spincore_IR_v1':    
+        s = find_file(searchstr, exp_type=exp_type,
             expno=nodename,
             postproc=postproc, lookup=postproc_dict,
-            dimname='indirect')
-    s *= exp(-1j*s.fromaxis('indirect')*clock_correction)
+            clock_correction=clock_correction, dimname='indirect')
```

Comment:
so you've added the clock correction to your preprocessing, and this gets passed to the preprocessing function?

Reply:
yes. This is in preprocessing for when applicable

Reply:
cool